### ISSUE TITLE
Inversion merge: Check for possible merging inversions every time inversion breakpoints are modified

### ISSUE THREAD

**Issue by cacereslab** [caceres-lab/InvFEST-code#28]
It would be nice if once the breakpoints of an inversion are modified, the database automatically searches for possible inversions with overlapping inversions an gives a warning (the user can decide if it merges the inversions or not using the merging tool).

**Comment by RuthGG** [comment]
I think that this has been somewhat solved; not exactly a warning but in the Advanced edition section when an inversion has >90% of reciprocal overlap it informs the user that it could be a good match for a merge. 

![image](https://user-images.githubusercontent.com/22767844/53168375-c5d93580-35da-11e9-86c7-d59baa097a54.png)

Maybe this message can be completed with more information about the criteria used to find the match, and another line can be incorporated with the inversions that match breakpont-to-breakpoint with the one in the Inversion Report.

**Comment by cacereslab** [comment]
Yes, maybe we could include below each candidate inversion to merge if the breakpoints match perfectly with those of the reported inversion and the % of reciprocal overlap between them.
